src/Microsoft.ApplicationInsights/Extensibility/TelemetryConfiguration.cs

```diff
@@ -234,6 +234,19 @@ public ITelemetryChannel TelemetryChannel
         /// </remarks>
         public IApplicationIdProvider ApplicationIdProvider { get; set; }
 
+        /// <summary>
+        /// Gets or sets a string array (string[]) indicating if an experimental feature should be enabled.
```

Comment:
nit: this is not an array (string [])

Reply:
an array is technically an IEnumerable, so it's not incorrect :)

Reply:
fixed